src/Metadata/ApiProperty.php

```diff
@@ -24,23 +30,24 @@
+     * @param bool|null                                                                $writableLink            https://api-platform.com/docs/core/serialization/#force-iri-with-relations-of-the-same-type-parentchilds-relations
+     * @param bool|null                                                                $required                https://api-platform.com/docs/admin/validation/#client-side-validation
+     * @param bool|null                                                                $identifier              https://api-platform.com/docs/core/identifiers/
+     * @param mixed                                                                    $example                 https://api-platform.com/docs/core/openapi/#using-the-openapi-and-swagger-contexts
+     * @param string|null                                                              $deprecationReason       https://api-platform.com/docs/core/deprecations/#deprecating-resource-classes-operations-and-properties
+     * @param bool|null                                                                $fetchEager              https://api-platform.com/docs/core/performance/#eager-loading
+     * @param array|null                                                               $jsonldContext           https://api-platform.com/docs/core/extending-jsonld-context/#extending-json-ld-and-hydra-contexts
+     * @param array|null                                                               $openapiContext          https://api-platform.com/docs/core/openapi/#using-the-openapi-and-swagger-contexts
+     * @param bool|null                                                                $push                    https://api-platform.com/docs/core/push-relations/
+     * @param string|\Stringable|null                                                  $security                https://api-platform.com/docs/core/security
+     * @param string|\Stringable|null                                                  $securityPostDenormalize https://api-platform.com/docs/core/security/#executing-access-control-rules-after-denormalization
+     * @param string[]                                                                 $types                   the RDF types of this property
+     * @param string[]                                                                 $iris
+     * @param Type[]                                                                   $builtinTypes
+     * @param string|null                                                              $uriTemplate             (experimental) whether to return the subRessource collection IRI instead of an iterable of IRI
+     * @param string|null                                                              $property                The property name
+     * @param array<int, Groups|SerializedName|SerializedPath|MaxDepth|Ignore|Context> $serialize
```

Comment:
```suggestion
     * @param array<int, Context|DiscriminatorMap|Groups|Ignore|MaxDepth|SerializedName|SerializedPath> $serialize
```

(`Context` and `DiscriminatorMap` are missing, and sort in alphabetical order)

Reply:
`DiscriminatorMap` is a `class` attribute only